# Calls up a function on the server node and returns the int return value
int8 function
string operands
---
int8 result